### PR TITLE
fixed swap issue

### PR DESCRIPTION
**Pull request by Emmah-lou**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,7 +16,9 @@
     </script>
     <!-- End Google Tag Manager -->
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="description" contnet="clothes swap app for lgbtq communinity, gay trans bisexual and lesbian resources" />
```
Comment (BlocksOnChainCode) [inline index.html:19]: Typo girl 😉

File: public/assets/react.svg
Comment (BlocksOnChainCode) [inline public/assets/react.svg:1]: Wondering why it shows like this?

File: src/components/home/HomeHeader.jsx [hunk 2]
```diff
@@ -10,12 +15,35 @@ const HomeHeader = () => {
...
+            setIsOpen(!isOpen);
+          }}
+        >
+          <BugReportModal
+            toggleModal={() => {
+              setIsOpen(!isOpen);
+            }}
+          />
+        </Modal>
+      )}
```
Comment (BlocksOnChainCode) [inline src/components/home/HomeHeader.jsx:46]: Good call!

File: src/hooks/useSwap.js [hunk 3]
```diff
@@ -18,7 +18,10 @@ const useSwap = () => {
       })
       .select();
     if (error) {
+      console.log(error);
       setSwapError(error.message);
+      alert('Swap failed');
+      return;
```
Comment (BlocksOnChainCode) [inline src/hooks/useSwap.js:24]: Quite high on the priority list for me, the error modal for providing proper user feedback.

**Review by BlocksOnChainCode — COMMENTED** [review]
💯

**Review by BlocksOnChainCode — APPROVED** [review]
💯

File: index.html (hunk 1 above)
Comment (Emmah-lou) [reply index.html:19]: contnet

File: public/assets/react.svg
Comment (Emmah-lou) [reply public/assets/react.svg:1]: we will have to play around with it, fill line woulndt change color

Comment (BlocksOnChainCode) [reply public/assets/react.svg:1]: I meant why it showed the 'react logo' was deleted and a clothing icon got there instead.

File: index.html (hunk 1 above)
Comment (BlocksOnChainCode) [reply index.html:19]: Oh! two typos then 😆

**Pull request merged**